bundle/src/assembly/resources/scenarios/Tiergarten/omnetpp/omnetpp.ini

```diff
@@ -61,10 +61,10 @@ Simulation.mgmt.cmdenv-log-level = info   # = info
 #**.wlan*.bitrate = 6Mbps
```

Comment:
have you found anything regarding this? do we need to configure the bitrate here? @fhilg @rprotzmann

Reply:
As discussed, the line is removed.